packages/ui/src/views/layouts/PullRequestLayout.tsx

```diff
@@ -14,72 +15,89 @@ interface PullRequestLayoutProps {
   useTranslationStore: () => TranslationStore
 }
 
+enum PullRequestTabsKeys {
+  CONVERSATION = 'conversation',
+  COMMITS = 'commits',
+  CHANGES = 'changes'
+}
+
+const pullRequestTabsKeysArr = Object.values(PullRequestTabsKeys)
+
 const PullRequestLayout: React.FC<PullRequestLayoutProps> = ({
   usePullRequestStore,
   useTranslationStore,
   spaceId,
   repoId
 }) => {
+  const location = useLocation()
   const { pullRequest } = usePullRequestStore()
   const { t } = useTranslationStore()
 
-  const baseClasses =
-    'inline-flex items-center justify-center px-3 py-1 px-4 items-center gap-2 bg-background hover:text-primary h-[36px] rounded-tl-md rounded-tr-md m-0'
-  const getLinkClasses = (isActive: boolean) => {
-    return `${baseClasses} ${isActive ? 'text-primary [&svg]:text-primary tabnav-active' : 'tabnav-inactive'}`
-  }
+  const activeTab = useMemo(() => {
+    if (location.pathname.includes(PullRequestTabsKeys.COMMITS)) {
```

Comment:
using `location` for determining active tab is a regression. react-router already provides the active path, which was being used earlier. 
why do we need to make this change?